### ISSUE TITLE
Recurrence code of SVHN original datasets

### ISSUE THREAD

**Issue by WangzekunY** [Bartzi/see#107]
When I get the original SVHN dataset from [here](http://ufldl.stanford.edu/housenumbers/)，I find train.tar.gz, test.tar.gz , extra.tar.gz. I want to reproduce the results with 95.2% accuracy obtained in the paper, I don't quite understand which dataset you use as a training set and a validation set

**Comment by Bartzi** [comment]
Sry, for the long time to answer. As far as I remember, We used the files `train` and `test` for training and testing. You can also split some parts of the train dataset to obtain a validation split. The `extra` file is not used.

**Comment by WangzekunY** [comment]
Thanks for your reply, could you provide  training parameters with reproduction accuracy？

**Comment by Bartzi** [comment]
Hmm, :sweat_smile: since we published this many years ago. I don't think I can give you the exact training parameters, but I can provide you with the [trained model](https://bartzi.de/research/stn-ocr) but be aware that this model does not work with the code from this repo. It only works with the code from our earlier [repo](https://github.com/Bartzi/stn-ocr) but this code is very very old. I'm not sure whether it is still easily possible to make it work...

Anyway, to train such a model on SVHN, you'll need to follow the steps for the training of text recognition models.
